src/pages/home/report/comment/AttachmentCommentFragment.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../../../styles/styles';
+import RenderCommentHTML from './RenderCommentHTML';
+
+const propTypes = {
+    /** The reportAction's source */
+    source: PropTypes.oneOf(['Chronos', 'email', 'ios', 'android', 'web', 'email', '']).isRequired,
```

Comment:
Where did you get this enum? Can we also create separate propTypes file to avoid duplicated definitions?

Reply:
Makes sense! Done.